wb/main/shared/enumerates.py

```diff
@@ -14,7 +14,9 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 """
+from __future__ import annotations
```

Comment:
remove

Reply:
This import is what allows for function output typing in line 63. Is it really necessary to remove this?